packages/dso-toolkit/src/components/logo/logo.stories-of.ts

```diff
@@ -0,0 +1,60 @@
+import { Addon_PartialStoryFn } from "@storybook/types";
+
+import { StoriesOfArguments, storiesOfFactory } from "../../storybook/index.js";
+
+import { LogoArgs, logoArgsMapper, logoArgTypes } from "./logo.args.js";
+import { Logo } from "./logo.models.js";
+
+export interface LogoTemplates<TemplateFnReturnType> {
+  logoTemplate: (logoProperties: Logo) => TemplateFnReturnType;
+}
+
+export interface LogoParameters<TemplateFnReturnType> {
+  decorator: (story: Addon_PartialStoryFn<TemplateFnReturnType>) => TemplateFnReturnType;
+}
+
+export function storiesOfLogo<Implementation, Templates, TemplateFnReturnType>(
+  storiesOfArguments: StoriesOfArguments<
+    Implementation,
+    Templates,
+    TemplateFnReturnType,
+    LogoTemplates<TemplateFnReturnType>
+  >
+) {
+  return storiesOfFactory("Logo", storiesOfArguments, (stories, templateMapper) => {
+    stories.addParameters({
+      argTypes: {
+        logoArgTypes,
+      },
+      args: {
+        label: "",
+        ribbon: "",
+      },
```

Comment:
In `LogoArgs` heb je netjes rekening gehouden dat `label` en `ribbon` optioneel zijn (met de `?` suffix), maar hier zet je een default value van een lege string.

Ik denk dat het fraaier is om deze default args weg te laten, wij hebben immers ondersteuning ingebouwd dat er géén label en/of ribbon worden meegegeven (niets is iets anders dan een lege string).

Reply:
Mee eens, ik verwijder ze.